.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -37,6 +37,10 @@ parameters:
   - name: windowsCoreImageUrl
     displayName: Windows core base container image URL Override
     type: string
+  - name: windowsContainerImageJsonUrl
+    displayName: Windows container image JSON URL Override
+    type: string
+    default: https://wcctagentbakerstorage.blob.core.windows.net/simship/2025/07B/payload.json
```

Comment:
And empty as well?  Just to make sure the default is what we want if this is consumed somewhere else